grpc/src/main/java/io/stargate/grpc/codec/cql/VarintCodec.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.grpc.codec.cql;
+
+import com.google.protobuf.ByteString;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.stargate.db.schema.Column.ColumnType;
+import io.stargate.proto.QueryOuterClass.BigInteger;
+import io.stargate.proto.QueryOuterClass.Value;
+import io.stargate.proto.QueryOuterClass.Value.InnerCase;
+import java.nio.ByteBuffer;
+
+public class VarintCodec implements ValueCodec {
+  @Override
+  public ByteBuffer encode(@NonNull Value value, @NonNull ColumnType type) {
+    if (value.getInnerCase() != InnerCase.VARINT) {
+      throw new IllegalArgumentException("Expected varint type");
+    }
+    BigInteger varint = value.getVarint();
+    return ByteBuffer.wrap(varint.getValue().toByteArray());
```

Comment:
we could leverage the java-driver codecs(`TypeCodecs.VARINT`):
```
  @Override
  public ByteBuffer encode(@NonNull Value value, @NonNull ColumnType type) {
    if (value.getInnerCase() != InnerCase.VARINT) {
      throw new IllegalArgumentException("Expected varint type");
    }
    BigInteger bigInteger = new BigInteger(value.getVarint().toByteArray());
    return TypeCodecs.VARINT.encode(bigInteger, PROTOCOL_VERSION);
  }
```
but It would mean that we need to create a `BigInteger` instance only for the purpose of encoding. I am not sure if that is a good value from the performance perspective. Instead, we can inline the logic and just return `ByteBuffer`.
Same for `decode(...)`. Wdyt?

Reply:
Seems like overkill for how simple these codecs are.